src/lib/variants.test.ts

```diff
@@ -90,8 +90,9 @@ describe('findTestAndVariant', () => {
 
         const got = findTestAndVariant(tests, targeting);
 
-        expect(got?.test.name).toBe('example-1');
-        expect(got?.variant.name).toBe('control-example-1');
+        // expect(got?.test.name).toBe('example-1');
+        // expect(got?.variant.name).toBe('control-example-1');
+        expect(got).toBe(undefined);
```

Comment:
I wonder if it would be worth keeping this test as one where we successfully match a test/variant (which I think would mean removing the `articlesViewedSettings` from the input test in this case) and perhaps adding another scenario where there _are_ `articlesViewedSettings` which causes an `undefined` result. What do you think?

Reply:
👍